components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -299,6 +356,230 @@ export default {
+        lines.push({
+          content: `Compared to the apex score of ${serverHighScore} achived by ${serverHighScoreUser}, their score can be improved. They should consider improving their agreeability in the new year.`
+        })
+      }
+
+      this.executeCommandText(lines)
+    },
+    SecBotScoresLow() {
+      const user = this.username
+      const userLowScore = this.stats.user['SecBot user min score'].score
+      const serverLow = this.stats.server['SecBot server low score']
+      const serverLowScore = serverLow.score
+      const serverLowScoreUser =
+        this.stats.server.idsToNames[serverLow.member_id]
+
+      const lines = [
+        { content: `Low Score`, class: 'underlined-text' },
+        { content: `${user}'s lowest score was ${userLowScore}.` }
+      ]
+
+      if (userLowScore === 750) {
+        lines.push({
+          content: `If the records are correct, ${user}'d score never dropped below their starting value, this behavior does not go unnoticed.'`
+        })
+      }
+
+      if (userLowScore === serverLowScore) {
+        lines.push({
+          content: `Incidently, this was the lowest score recorded since SecurityBot's inception on 4/1/22. The associated behavior that resulted in this score has hopefully been corrected.`
+        })
+      } else {
+        lines.push({
+          content: `Even this score can be made to look impressive compared to the lowest recorded score of ${serverLowScore} by ${serverLowScoreUser}.`
+        })
+      }
+      this.executeCommandText(lines)
+    },
+    SecBotScoresAverage() {
+      const user = this.username
+      const userAverage = this.stats.user['SecBot user average score'].score
+      const serverAverage = 55 // this.stats.server['']
+
+      const lines = [
+        { content: `Average Scores`, class: 'underlined-text' },
+        {
+          content: `The average score of the entire server over the year was ${serverAverage}`
+        }
+      ]
+
+      if (userAverage >= serverAverage) {
+        lines.push({
+          content: `${user}'s average score of ${userAverage}, on average, was higher on average than the average, on average.`
+        })
+      } else {
+        lines.push({
+          content: `${user}'s average score of ${userAverage}, on average left a lot to be desired, on average.`
+        })
+      }
+
+      this.executeCommandText(lines)
+    },
+    MetaSecBotDevelopment() {
+      // describe how the scoring worked? including the initial method and how it was changed to be wackier after a few weeks while ben was drunk? (cheeto cheeto)
+      // include how scores loop around from 1600
+      // at 7am on april 1st Ethan and Ben met up to make sure that it worked, and it didnt so we needed to manually activate it.
+      // creation of Dennis
+      // CUMPEG
+      // format all of these like they are chat logs between Ben and Ethan?
+      // have some corrupted / jumbled text
+      // folder title: subpoena documents, backups,
```

Comment:
I feel like this file should be re-composed into smaller chunks ASAP

Reply:
a low effort possibility would be to extract the functions out into their own file and import it. maybe the file also deserves a rename if its scope has changed

Reply:
tbf tho this is a problem on the `Chat` side as well - `Text.vue` includes mappings of textual representation of data to the actual data itself. however, Text.vue doesn't actually contain or instantiate any data/text - it just maps from a key to a value.